playbooks/roles/deploy-osh/tasks/main.yml

```diff
@@ -417,3 +417,20 @@
   tags:
     - heat
     - run
+
+- name: Deploy Tempest
+  import_tasks: component-install.yml
+  vars:
+    _component_runcommand: "./tools/deployment/multinode/900-tempest.sh"
+    _component_run_env:
+      OSH_EXTRA_HELM_ARGS_HEAT: >
```

Comment:
Is `HEAT` correct here? I don't understand it fully but just checking that it isn't a copy&paste error.

Reply:
It probably is :)

Reply:
woop woop, you didnt see anything